packages/canvas-extract/src/CanvasExtract.ts

```diff
@@ -63,6 +63,20 @@ export class CanvasExtract implements ISystem, IExtract
     {
         const canvas = this.canvas(target);
 
+        if (canvas.toBlob !== undefined)
+        {
+            return await new Promise<string>((resolve, reject) =>
```

Comment:
The await here is unnecessary and creates an additional promise. We should remove it.

```suggestion
            return new Promise<string>((resolve, reject) =>
```